src/types.ts

```diff
@@ -325,6 +327,7 @@ export interface WebAppPrefs {
   brandLogoFileId?: string
   brandLogoOnDarkFileId?: string
   defaultListFilters?: string[]
+  userRegistrationCode?: string
```

Comment:
Small change to front-end (already merged) -- allows us to specify the code of the special "user registration" form rather than having it hard-coded (still has same default of `UserRegistration`, but I've changed it to `demoRegister` for the demo system)